odoo/addons/test_search_panel/tests/test_search_panel_select_range.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+import odoo.tests
+
+
+@odoo.tests.tagged('post_install', '-at_install', 'dam')
+class TestSelectRange(odoo.tests.SavepointCase):
```

Comment:
why not odoo.tests.common.TransactionCase ?

Reply:
I thought this was the root cause of my 'problem' with Ids growing...
This was the case before.
Thx

Reply:
done